configure.ac

```diff
@@ -175,7 +182,7 @@ fi
 
 # Check for Python modules
 AX_PYTHON_MODULE(numpy, fatal)
-AX_PYTHON_MODULE(bigfloat, fatal)
+AX_PYTHON_MODULE(bigfloat)
```

Comment:
This seems a good idea, since `bigfloat` is only a dependency for tests.

Reply:
same as openmp, this is why I was wondering libomp this shouldn't be added to dependency list. On the other hand as we discussed monday, a mechanism to disable the test/part of the test concerned by a given dependency might be a good idea.